app/bundles/CampaignBundle/Entity/LeadEventLog.php

```diff
@@ -115,6 +115,8 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->addIndex(['date_triggered'], 'campaign_date_triggered')
             ->addIndex(['lead_id', 'campaign_id', 'rotation'], 'campaign_leads')
             ->addIndex(['channel', 'channel_id', 'lead_id'], 'campaign_log_channel')
+            ->addIndex(['event_id', 'non_action_path_taken'], 'campaign_actions')
+            ->addIndex(['campaign_id', 'event_id', 'date_triggered'], 'campaign_stats')
```

Comment:
This likely needs non_action_path_taken included for the index to be picked up with a large cardinality (without forcing/hinting the indeces).